robottelo/ui/container.py

```diff
@@ -0,0 +1,160 @@
+# -*- encoding: utf-8 -*-
+from robottelo.constants import FILTER
+from robottelo.ui.base import Base, UINoSuchElementError, UIError
+from robottelo.ui.locators import locators, tab_locators
+from robottelo.ui.navigator import Navigator
+
+
+class Container(Base):
+    """Provides the CRUD functionality for Docker Containers."""
+
+    def _configure_orgs(self, orgs, org_select):
```

Comment:
I'm wondering if `_configure_orgs` and `_configure_locations` shouldn't be merged into a single method, since they are very, very similar?